src/screens/main/story/StoryScreen.tsx

```diff
@@ -218,6 +262,81 @@ const StoryScreen: React.FC<NativeStackScreenProps<MainStackParams, typeof Scree
 };
 
 const styles = StyleSheet.create({
+    wrapper: {
+        flex: 1,
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        justifyContent: 'space-between',
+        backgroundColor: colors.WHITE,
+    },
+    content__profile: {
+        width: 24,
+        height: 24,
+    },
+    content__cover: {
+        height: '100%',
+        borderRadius: 16,
+    },
+    content__profile_wrapper: {
+        alignItems: 'flex-end',
+    },
+    content__book_cover: {
+        alignItems: 'flex-start',
+    },
+    content__boxpic: {
+        alignItems: 'flex-start',
+        width: 24,
+        height: 24,
+    },
+    content__header: {
+        fontSize: sizes.TEXT_BIG,
+        fontFamily: 'RFDewi_Bold',
+        marginBottom: sizes.PADDING_LITTLE,
+    },
+    content__cover_boxes: {
+        flexDirection: 'row',
+        alignItems: 'center',
+        height: 250,
+        justifyContent: 'space-between',
+    },
+    content__white_box: {
+        width: 38,
+        height: 38,
+        backgroundColor: colors.WHITE,
+        borderRadius: 8,
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    content__boxg: {
+        marginTop: sizes.PADDING_SMALL,
+        backgroundColor: colors.GREY_BOX,
+        borderRadius: 12,
+        height: 34,
+        width: Dimensions.get('window').width / 3.7,
```

Comment:
Magic numbers again. If you want to do some hardcode, to better write *0.27 (~1/3.7) insted of /3.7, because it event doesn't make sense when someone else looks at this